doc/api/events.md

```diff
@@ -103,6 +107,8 @@ listener functions can switch to an asynchronous mode of operation using
 the `setImmediate()` or `process.nextTick()` methods:
 
 ```js
+'use strict';
+
```

Comment:
This example both works without `use strict` and is partial (a continuation of the previous example), that means that this is not the correct place for `use strict`.